artists#logical/tic_tac_toe_improved.rb

```diff
@@ -0,0 +1,159 @@
+require 'colorize'
+#Variables generales
```

Comment:
Se acostumbra poner un espacio después de `#`

Reply:
Can you explain this line to me? what does colorize gem do?